src/server/load_statistics.rs

```diff
@@ -0,0 +1,149 @@
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::sync::Arc;
+use std::time::Instant;
+
+use libc::{getpid, pid_t};
+
+use util::metrics::{get_thread_ids, Stat};
+
+pub struct ThreadLoad {
```

Comment:
Please add comments for each member.

Reply:
They are private, and there are already rust-docs for method `term()` and `load()`. So let's keep it clean.